content/en/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -227,6 +227,30 @@ can happen, according to:
 - the type of controller
 - the cluster's resource capacity
 
+## Pod disruption conditions {#pod-disruption-conditions}
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+When you enable the `PodDisruptionsCondition` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+in your cluster, a dedicated Pod `DisruptionTarget` condition is added to indicate
```

Comment:
Worth highlighting this section (the need to enable the feature gate) in a `note`.

Reply:
Moved into note